docs/js-api-reference.rst

```diff
@@ -0,0 +1,29 @@
+JavaScript API Reference
+========================
+
+Following are the functions that are defined in `rtd_sphinx_search.js`_,
+You can overwrite any of them, if any kind of customizations are required.
```

Comment:
How can you overwrite them? Would be useful to have a quick example perhaps, if we want folks to do it.

Reply:
Shouldn't this be "override".